firebase-inappmessaging/src/main/java/com/google/firebase/inappmessaging/internal/InAppMessageStreamManager.java

```diff
@@ -233,7 +234,7 @@ private boolean shouldIgnoreCache(String event) {
 
               Function<CampaignImpressionList, Maybe<FetchEligibleCampaignsResponse>> serviceFetch =
                   impressions ->
-                      Maybe.fromCallable(() -> apiClient.getFiams(impressions))
+                      taskToMaybe(apiClient.getFiams(impressions))
```

Comment:
should this be task to single? http://reactivex.io/RxJava/3.x/javadoc/io/reactivex/rxjava3/core/Single.html

Reply:
next cl